CODEOWNERS

```diff
@@ -18,11 +18,9 @@
 /test-manifests/99-default-test-nrf.yml     @thst-nordic
 
 # Github Actions
-/.github/                                 @thst-nordic
-/.github/labeler.yml                      @thst-nordic @carlescufi @datgizmo
-/.github/mergify.yml                      @thst-nordic @carlescufi @datgizmo
+/.github/                                 @nrfconnect/ncs-ci
```

Comment:
@gmarull should be here as well

Reply:
As discussed Gerard can be added to ncs-ci. I removed doc* line.